packages/fluentui/react-northstar/src/themes/teams/components/Chat/chatMessageReadStatusIndicatorVariables.ts

```diff
@@ -0,0 +1,11 @@
+import { pxToRem } from '../../../../utils';
+
+export interface ChatMessageReadStatusIndicatorVariables {
+  readStatusIndicatorRightPoistion?: string;
+  readStatusIndicatorBottomPoistion?: string;
+}
+
+export const chatMessageReadStatusIndicatorVariables = (siteVars): ChatMessageReadStatusIndicatorVariables => ({
+  readStatusIndicatorRightPoistion: pxToRem(-24),
```

Comment:
I would remove the `readStatusIndicator` prefix, as all these variables are for the same component.

Reply:
Done